app/lib/features/chat_ng/widgets/events/chat_event_item.dart

```diff
@@ -124,6 +135,7 @@ class ChatEventItem extends StatelessWidget {
           messageId: messageId,
           content: content,
         ),
+      'm.location' => LocationMessageEvent(content: content),
```

Comment:
I think I mentioned before that I prefer to _not_ include support for items if the support is bad. This doesn't seem like a good way to support this. I'd rather skip supporting `m.location` and show a "not supported" message than a badly supported one...